splunk-forwarder/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "filter_patterns" {
+  description = "The filter expression watch filter for the subscription "
+  type        = "list"
+  default     = []
+}
+
+variable "log_group_names" {
+  description = "The log group"
+  type        = "list"
+  default     = []
+}
+
+variable "lambda_timeout_seconds" {
+  description = "Timeout for the lambda function that copies log entries to cloudwatch. Modify if you experience timeouts "
+  default     = 60
+}
+
+variable "lambda_memory_size" {
+  description = "Memory allocation for the lambda function that copies log entries to cloudwatch. Modify if you experience out of memory errors"
+  default     = 512
+}
+
+variable "lambda_bucket" {
+  default     = "telia-common-logs-prod-lambda"
```

Comment:
These defaults should be removed in my opinion, since it's an open source repository.

Reply:
I think we can use it if its a public bucket, otherwise we should remove the default.

Reply:
For this variable, that is true. Still I think it encourages a "bad practice" for 3rd parties though; running code that was generated by another 3rd party user without having access to the source code 😛

Reply:
I suggest that we link to the public repository where the Java Code is, so anyone can build it and upload to their own S3 bucket.

Reply:
That sounds great 😄